### PR TITLE
Session Task

### PR DESCRIPTION
**Pull request by RuiAAPeres**

### CODE REVIEW

**Comment by RuiAAPeres** [comment]
closes #13

**Comment by dmcrodrigues** [comment]
🤓

File: Vinyl/URLSessionTask.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    
+    init(completion: Void -> Void) {
+        self.completion = completion
+    }
+    
+    override func resume() {
+        completion()
+    }
+    
+    override func cancel() {
```
Comment (dmcrodrigues) [inline Vinyl/URLSessionTask.swift]: It's very unlikely but a caller may call `-suspend` and then... :boom: 

Should we also override that method?

**Comment by dmcrodrigues** [comment]
Great job :sparkles:

**Comment by dmcrodrigues** [comment]
:tractor:

**Pull request merged**